doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -285,6 +285,7 @@ Bluetooth samples
 * :ref:`peripheral_fast_pair` sample:
 
   * Updated by disabling the :kconfig:option:`CONFIG_BT_SETTINGS_CCC_LAZY_LOADING` Kconfig option as a workaround fix for the `Zephyr issue #61033`_.
```

Comment:
Should we add an entry for the adv_prov subsystem (about increased buffer configuration)?

Reply:
Hmm I think we should. I will add it. However, I would rather not add known issue for adv_prov subsystem as it's only matter of allowed range. Are you ok with that?

Reply:
sure, one known issue seems enough

Reply:
Old range does not allow value of 26 (needed when battery data is included in Fast Pair advertising and 10 Account Keys have been written). Maybe we could add a known issue under the Fast Pair advertising provider instead and link it in the sample's section?

Reply:
Those are two separate issues. First one is choosing appropriate size for sample. Second one is allowing appropriate range in Advertising Providers. I can add them both but I would rather not link them to not mix it up.

Reply:
My initial thought was to not add the range issue as I think it's neglectable, but I can add it if you insist.

Reply:
Definitely this should be one known issue, not two. Still, it's an issue of an advertising data provider, not sample itself (currently sample does not even configure this option), so I would put it under the advertising data provider's section (if you think it's needed, we could also link it under sample's section). Then you could merge the fix first and then link commit that needs to be cherry-picked to fix the issue as workaround.

Reply:
I disagree. It's sample responsibility to set appropriate Kconfig value. Using default value is also a choice.

Reply:
Discussed offline. Agreed to move the known issue under the advertising provider.